src/components/Layout/Layout.jsx

```diff
@@ -7,21 +7,26 @@ import { Banner } from '../Banner';
 
 export const Layout = ({ children, showSideBar }) => {
   return (
-    <>
+    <div style={{ height: '100vh', display: 'grid', gridTemplateRows: 'min-content min-content 1fr' }}>
       <NavigationBar />
-      <Container maxWidth="xl">
-        {showSideBar ? (
-          <Grid container>
-            <Banner />
-            <Sidebar />
-            <Box component="main">{children}</Box>
-          </Grid>
-        ) : (
+      {showSideBar ? (
+        <>
+          <Banner />
+          <Container maxWidth="xl" style={{ display: 'grid' }}>
+            <Grid container>
+              <Sidebar />
+              <Box component="main">{children}</Box>
+            </Grid>
+          </Container>
+        </>
+      ) : (
+        <Container maxWidth="xl" style={{ display: 'grid' }}>
           <Grid container justifyContent="center">
             <Box component="main">{children}</Box>
           </Grid>
-        )}
-      </Container>
-    </>
+        </Container>
+      )}
+    </div>
   );
 };
+//DDFDSAA
```

Comment:
to remove?

Reply:
xD

Reply:
this is a very important comment in project 🙂